nextflow.config

```diff
@@ -157,13 +158,14 @@ profiles {
         executor.cpus          = 16
         executor.memory        = 60.GB
     }
-    test               { includeConfig 'conf/test.config'               }
-    test_mouse         { includeConfig 'conf/test_mouse.config'         }
-    test_tiny          { includeConfig 'conf/test_tiny.config'          }
-    test_assembly_only { includeConfig 'conf/test_assembly_only.config' }
-    test_bins          { includeConfig 'conf/test_bins.config'          }
-    test_coassembly    { includeConfig 'conf/test_coassembly.config'    }
-    test_full          { includeConfig 'conf/test_full.config'          }
+    test                    { includeConfig 'conf/test.config'                    }
+    test_mouse              { includeConfig 'conf/test_mouse.config'              }
+    test_tiny               { includeConfig 'conf/test_tiny.config'               }
+    test_assembly_only      { includeConfig 'conf/test_assembly_only.config'      }
+    test_bins               { includeConfig 'conf/test_bins.config'               }
+    test_coassembly         { includeConfig 'conf/test_coassembly.config'         }
+    test_full               { includeConfig 'conf/test_full.config'               }
+    test_allele_information { includeConfig 'conf/test_allele_information.config' }
```

Comment:
Would `*model_information` be more precise than `*allele_information`?

Reply:
(i.e. it's information about the available models for each allele and length ... maybe just a different view point)

Reply:
Yes true, but as we are going with show_model_information the test should be named with model as well.